R/convertGlobalEnergyMonitor.R

```diff
@@ -0,0 +1,25 @@
+#' Convert Global Energy Monitor data
+#'
+#' @md
+#' @param x A [`magpie`][magclass::magclass] object returned from
+#'          [`readGlobalEnergyMonitor()`].
+#'
+#' @return A [`magpie`][magclass::magclass] object.
+#'
+#' @author Falk Benke
+#'
+#' @importFrom madrat toolCountry2isocode toolCountryFill
+#'
+#' @export
+convertGlobalEnergyMonitor <- function(x) {
+  mapping <- c(
+    "Åland Islands" = "ALA",
+    "DR Congo" = "COD",
+    "Republic of the Congo" = "COG",
+    "The Gambia" = "GMB",
+    "Virgin Islands (U_S_)" = "VIR"
+  )
```

Comment:
why did you add this mapping to the function and did not extend the mapping of the package, used in toolCountry2isocode()?